highfidelity/hifi
# Audio devices split

https://highfidelity.fogbugz.com/f/cases/edit/6885/Split-Desktop-and-VR-Audio-Settings

Test plan:
- open Tablet in Desktop mode
- click Audio button
- try to switch audio devices for both Desktop/VR contexts
- make sure it works
- switch to VR/HMD
- make sure it switched to device, selected previously in Desktop mode
- repeat pp. 2-4
- Make sure audio settings are saved between sessions
